doc/2/api/controllers/auth/create-api-key/index.md

```diff
@@ -6,7 +6,11 @@ title: createApiKey
 
 # createApiKey
 
-Creates a new API key for the currently loggued user.
+Creates a new API key for the currently logged user.
+
+::: info
+You can use your API key the same way you use your auth token, either by adding it to the `Authorization` header (with the value `Bearer <YourApiKeyHere>`) or by adding it to the query parameters of the HTTP call of an action (`?jwt=<YourApiKeyHere>`)
```

Comment:
```suggestion
You can use your API key the same way you use your auth token, either by adding it to the `Authorization` header (with the value `Bearer <YourApiKeyHere>`)
```
Putting the JWT in the the URL is a bad practice because it appears in the access logs